engine/src/test/java/org/entando/entando/web/page/validator/PageValidatorTest.java

```diff
@@ -0,0 +1,102 @@
+package org.entando.entando.web.page.validator;
+
+import com.agiletec.aps.system.services.page.IPage;
+import com.agiletec.aps.system.services.page.IPageManager;
+import com.agiletec.aps.system.services.user.User;
+import com.agiletec.aps.system.services.user.UserDetails;
+import java.util.HashMap;
+import java.util.Map;
+import liquibase.pro.packaged.U;
+import org.assertj.core.util.Maps;
+import org.entando.entando.aps.system.services.page.PageAuthorizationService;
+import org.entando.entando.aps.system.services.widgettype.WidgetType;
+import org.entando.entando.aps.system.services.widgettype.WidgetTypeManager;
+import org.entando.entando.web.common.exceptions.ValidationGenericException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.DataBinder;
+import org.springframework.validation.Errors;
+
+@ExtendWith(MockitoExtension.class)
+class PageValidatorTest {
+
+    private UserDetails user = new User();
```

Comment:
This seems unused, since you are using `getMockUser()`.

Reply:
removed